libgoal/libgoal.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/algorand/go-algorand/config"
 	"github.com/algorand/go-algorand/crypto"
 	"github.com/algorand/go-algorand/daemon/algod/api/spec/common"
-	"github.com/algorand/go-algorand/daemon/algod/api/spec/v1"
+	v1 "github.com/algorand/go-algorand/daemon/algod/api/spec/v1"
```

Comment:
is this necessary?

Reply:
this is done by vscode's go linter on save, and I did not find a way to disable this check